derrida/books/management/commands/instance_data.py

```diff
@@ -98,6 +109,8 @@ def parse_date_certainty(self, instance):
     def instance_data(self, instance):
         '''Generate a dictionary of data to export for a single
          :class:`~derrida.books.models.Instance` object'''
+        
+        author_type = CreatorType.objects.get(name='author')
```

Comment:
It looks like you're fetching this once for each item; should probably fetch once and store on the class.

Reply:
✅ 3dae88c

Reply:
If you place it in on the class, you need to make sure this fixture is prepared every time the command is initialized (and I'm also having trouble with pytest db access.) This simple change is causing a lot of problems. I'm removing it and putting it aside for now, but if you feel strongly or have a proposed solution, I'm happy to return to it.

Reply:
Oh, right. We don't have to get the type first, we can just filter on type__name. Will that work without causing test problems? It shouldn't require a fixture that way.

Reply:
✅ 3af7c77d8facfe1286e316e5df3006e6f715fe52
Smart! thank you